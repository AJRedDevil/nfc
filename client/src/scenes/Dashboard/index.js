// npm packages
import React from 'react';

// our packages
import ClassicTable from './components/ClassicTable';
import H2HTable from './components/H2HTable';

const Dashboard = () => (
  <div>
    <ClassicTable />
    <H2HTable />
  </div>
);

export default Dashboard;
