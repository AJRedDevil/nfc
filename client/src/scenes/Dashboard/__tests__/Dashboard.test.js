// npm packages
import React from 'react';
import renderer from 'react-test-renderer';
import configureMockStore from 'redux-mock-store';
import {Provider} from 'react-redux';

// our packages
import Dashboard from '../index';
import DashboardMock from './mock';

// create mock store
const mockStore = configureMockStore();

test('# Dashboard', () => {
  const store = mockStore({
    ClassicStandings: DashboardMock.ClassicStandings,
    H2HStandings: DashboardMock.H2HStandings,
  });
  const wrapper = renderer
    .create(
      <Provider store={store}>
        <Dashboard store={store} />
      </Provider>
    )
    .toJSON();
  expect(wrapper).toMatchSnapshot();
});
