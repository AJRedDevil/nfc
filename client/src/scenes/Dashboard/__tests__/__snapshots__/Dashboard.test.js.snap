// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`# Dashboard 1`] = `
<div>
  <div>
    <section
      className="hero is-small is-info is-bold"
    >
      <div
        className="hero-body"
      >
        <div
          className="container"
        >
          <h1
            className="title"
          >
            test
          </h1>
          <h2
            className="subtitle"
          >
            testing
          </h2>
        </div>
      </div>
    </section>
    <table
      className="table is-fullwidth is-striped is-narrow is-hoverable"
    >
      <thead>
        <tr>
          <th>
            Rank
          </th>
          <th>
            Team
          </th>
          <th>
            Player
          </th>
          <th>
            Score
          </th>
        </tr>
      </thead>
      <tfoot>
        <tr />
      </tfoot>
      <tbody>
        <tr>
          <th>
            1
          </th>
          <td>
            Test_2
          </td>
          <td>
            Test 2
          </td>
          <td>
            100
          </td>
        </tr>
        <tr>
          <th>
            2
          </th>
          <td>
            Test_1
          </td>
          <td>
            Test 1
          </td>
          <td>
            70
          </td>
        </tr>
        <tr>
          <th>
            3
          </th>
          <td>
            Test_3
          </td>
          <td>
            Test 3
          </td>
          <td>
            60
          </td>
        </tr>
      </tbody>
    </table>
  </div>
  <div>
    <section
      className="hero is-small is-info is-bold"
    >
      <div
        className="hero-body"
      >
        <div
          className="container"
        >
          <h1
            className="title"
          >
            test
          </h1>
          <h2
            className="subtitle"
          />
        </div>
      </div>
    </section>
    <table
      className="table is-fullwidth is-striped is-narrow is-hoverable"
    >
      <thead>
        <tr>
          <th>
            Div
          </th>
          <th>
            Team
          </th>
          <th>
            Player
          </th>
          <th>
            Score
          </th>
        </tr>
      </thead>
      <tfoot>
        <tr />
      </tfoot>
      <tbody>
        <tr>
          <th>
            A
          </th>
          <td>
            Test_2
          </td>
          <td>
            Test 2
          </td>
          <td>
            100
          </td>
        </tr>
        <tr>
          <th>
            B
          </th>
          <td>
            Test_1
          </td>
          <td>
            Test 1
          </td>
          <td>
            70
          </td>
        </tr>
        <tr>
          <th>
            C
          </th>
          <td>
            Test_3
          </td>
          <td>
            Test 3
          </td>
          <td>
            60
          </td>
        </tr>
      </tbody>
    </table>
  </div>
</div>
`;
